src/RpcServer/RpcServer.Utilities.cs

```diff
@@ -36,7 +36,7 @@ protected virtual JToken ListPlugins(JArray _params)
         [RpcMethod]
         protected virtual JToken ValidateAddress(JArray _params)
         {
-            string address = _params[0].AsString();
+            string address = Result.Ok_Or(() => _params[0].AsString(), RpcError.InvalidParams.WithData($"Invlid address format: {_params[0]}"));
```

Comment:
We return null bellow when is invalid

Reply:
Only if it's a proper string. But if it's not a string, then it's impossible to check its validness, so I think it's OK to return error in this case.